doctor_visits/delphi_doctor_visits/run.py

```diff
@@ -61,30 +64,30 @@ def run_module(params):
     startdate_dt = enddate_dt - timedelta(days=n_backfill_days)
     enddate = str(enddate_dt.date())
     startdate = str(startdate_dt.date())
-    logging.info("drop date:\t\t%s", dropdate)
-    logging.info("first sensor date:\t%s", startdate)
-    logging.info("last sensor date:\t%s", enddate)
-    logging.info("n_backfill_days:\t%s", n_backfill_days)
-    logging.info("n_waiting_days:\t%s", n_waiting_days)
+    logger.info("drop date:\t\t%s", dropdate)
+    logger.info("first sensor date:\t%s", startdate)
+    logger.info("last sensor date:\t%s", enddate)
+    logger.info("n_backfill_days:\t%s", n_backfill_days)
+    logger.info("n_waiting_days:\t%s", n_waiting_days)
```

Comment:
does structlog support the `(str, *args)` signature? I would've expected these to need to be converted to either
```
logger.info(f"n_waiting_days:\t{n_waiting_days}")
```

or something like
```
logger.info("n_waiting_days", value=n_waiting_days)
```

Reply:
Seems to be work:
```
In [14]: import logging

In [15]: from delphi_utils import get_structured_logger

In [16]: logging_logger = logging.getLogger()

In [17]: struct_logger = get_structured_logger()

In [18]: geo = "test_geo"

In [19]: logging_logger.info("starting %s, no adj", geo)
starting test_geo, no adj

In [20]: struct_logger.info("starting %s, no adj", geo)
{"event": "starting test_geo, no adj", "logger": "delphi_utils.logger", "level": "info", "timestamp": "2021-09-25T03:03:54.217027Z"}
```